.circleci/config.yml

```diff
@@ -24,7 +24,7 @@ jobs:
           command: /saltyrtc/certs/generate-cert.sh
       - run:
           name: Copy certificate
-          command: cp /saltyrtc/certs/saltyrtc.der .
+          command: cp /saltyrtc/certs/saltyrtc.crt .
```

Comment:
Don't have a strong opinion about it but I'd usually use the `.pem` ending.

Reply:
True, but `crt` is more consistent with the other clients' test instructions.